setup.cfg

```diff
@@ -102,23 +112,35 @@ markers =
 remote_data_strict = True
 
 [pycodestyle]
-max_line_length = 100
+max_line_length = 110
 
 [flake8]
-max-line-length = 100
+max-line-length = 110
+exclude =
+    .git,
+    __pycache__,
+    docs/conf.py,
+    build,
+    sunpy/data/sample.py,
+    sunpy/extern/distro.py,
```

Comment:
can we ignore a whole directory?

Reply:
That makes more sense, I will check and add it.